pkg/controller/node/node_controller_test.go

```diff
@@ -325,6 +380,23 @@ func TestGetPoolForNode(t *testing.T) {
 
 func intStrPtr(obj intstr.IntOrString) *intstr.IntOrString { return &obj }
 
+func newMixedNodeSet(setlen int, roles ...map[string]string) []*corev1.Node {
```

Comment:
could we add a 1 liner comment about what these new helpers are?

Reply:
yup I'll add it

Reply:
thanks!

Reply:
fixed!